core/src/models/GrouparooModel.ts

```diff
@@ -90,13 +90,6 @@ export class GrouparooModel extends LoggedModel<GrouparooModel> {
     return instance;
   }
 
-  @BeforeCreate
-  static generateId(instance: GrouparooModel) {
-    if (!instance.id) {
-      instance.id = `${instance.idPrefix()}_${uuid.v4()}`;
-    }
-  }
```

Comment:
This was redundant with the method within `LoggedModel` that GrouparooModel extends